getting-started/README.adoc

```diff
@@ -170,6 +171,60 @@ ifndef::qs[]
 . Verify that your service account was successfully created in the *Streams for Apache Kafka* > *Service Accounts* page.
 endif::[]
 
+[id="proc-setting-service-account-permissions_{context}"]
+== Setting permissions for a service account in a Kafka instance in {product}
+
+{product} uses Access Control Lists (ACLs) provided by Kafka that enable you to manage how other user accounts and service accounts are permitted to interact with the Kafka resources that you create. When you create a service account to connect to a Kafka instance, you must also set the appropriate level of permissions for that new account in the ACL of the instance.
+
+The permissions described in this procedure enable applications associated with the service account to fully interact with the resources in the instance.
+
+.Prerequisites
+* You've created a Kafka instance and the instance is in *Ready* state.
+* You've created a service account that you want to allow to interact with the running Kafka instance.
```

Comment:
created service account doesnt interact with kafka. It is used as credentials when making api call.

Best to say service account that will be used when connecting to Kafka from your app or service

Reply:
Oops, yep. I went around the block on that wording and missed that.

How about this? This is more consistent with what we say elsewhere and more accurate. Work for you?

>You've created a service account that you want to allow to access the running Kafka instance.